### ISSUE TITLE
Command failed with exit code 129: git -P shortlog -nes -- README.md

### ISSUE THREAD

**Issue by bestvist** [vuepress/vuepress-next#16]
It's reason for 
```
 const { stdout } = await execa(
    'git',
    ['-P', 'shortlog', '-nes', '--', filePath],
    {
      cwd,
      stdin: 'inherit',
    }
  )
```

[https://github.com/vuepress/vuepress-next/blob/main/packages/%40vuepress/plugin-git/src/utils/getContributors.ts](https://github.com/vuepress/vuepress-next/blob/main/packages/%40vuepress/plugin-git/src/utils/getContributors.ts)

cause `-P` ?

**Comment by meteorlxy** [comment]
Could you provide more context?

- os
- shell
- git version
- steps to reproduce

This command has not been fully tested, and it might fail in some environments. But it will not help if you don't provide any context.

**Comment by bestvist** [comment]
My context: 
* MacOS
* vscode
* git@2.10.0

I'm use GitLab by company, cause this？

**Comment by meteorlxy** [comment]
Try to upgrade your git.

`-P` is available after 2.18.0

https://git-scm.com/docs/git/2.18.0#Documentation/git.txt--P
